packaging/RPMS/Fedora/rabbitmq-server.spec

```diff
@@ -11,6 +11,7 @@ Source2: rabbitmq-script-wrapper
 Source3: rabbitmq-server.logrotate
 Source4: rabbitmq-server.ocf
 Source5: README
+Source6: rabbitmq-server-ha.ocf
```

Comment:
This seems a cosmetic fix, although I agree with you. Let's keep this for the next change set please